Eyes.Common.DotNet/VisualGrid/IRenderRequest.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Applitools.VisualGrid
+{
+    public interface IRenderRequest
```

Comment:
Remove all interfaces that you can

Reply:
done, but this one is needed since it's implemented outside that project. Once I'll re-merge the Common and Sdk.Core projects I'll remove it.